packages/migration/migrations/hearth/20221017133354-restore-supporting-document.ts

```diff
@@ -24,13 +25,13 @@ export const up = async (db, client) => {
     await session.withTransaction(async () => {
       const totalCompositionCount = await getBirthEncounterCompositionCount(db)
       while (totalCompositionCount > processedDocCount) {
-        const compositionCursor = await getBirthEncounterCompositionCursor(
-          db,
-          limit,
-          skip
-        )
+        const compositionCursor =
+          await getBirthEncounterCompositionCursor<fhir.Composition>(
```

Comment:
```suggestion
          await getBirthEncounterCompositionCursor<fhir.Composition & { id: string}>(
```
This should enable you to not use "any" when getting the value from it